wdl/TasksMakeCohortVcf.wdl

```diff
@@ -1004,5 +1004,83 @@ task ScatterVcf {
   >>>
   output {
     Array[File] shards = glob("~{prefix}.shard_*.vcf.gz")
+    Array[String] shards_string = glob("~{prefix}.shard_*.vcf.gz")
+  }
+}
+
+task FixEndsRescaleGQ {
```

Comment:
What happened here?

Reply:
ping @alsanju

Reply:
I think this task was created but never used; maybe it is a dev leftover. We can delete it.

Reply:
This WDL is from GATK-SV repo and was included here for the SplitVCF task only, FixEndsRescaleGQ is not used and can be deleted